src/development/LocalTest/Services/Profile/Implementation/UserProfilesWrapper.cs

```diff
@@ -35,6 +38,11 @@ public async Task<UserProfile> GetUser(int userId)
                 user = (UserProfile)JsonConvert.DeserializeObject(content, typeof(UserProfile));
             }
 
+            if(!string.IsNullOrWhiteSpace(STATIC_LANGUAGE_OVERRIDE))
+            {
+                user.ProfileSettingPreference.Language = STATIC_LANGUAGE_OVERRIDE;
```

Comment:
LGTM seem to be picky about nulls in deserialized json 
```suggestion
                user.ProfileSettingPreference ??= new();
                user.ProfileSettingPreference.Language = STATIC_LANGUAGE_OVERRIDE;
```